jobs/resource_job_test.go

```diff
@@ -2436,21 +2442,14 @@ func TestResourceJobUpdate_NodeTypeToInstancePool(t *testing.T) {
 		Update:   true,
 		Resource: ResourceJob(),
 		InstanceState: map[string]string{
-			"new_cluster.0.node_type_id":                      "node-type-id-worker",
-			"new_cluster.0.driver_node_type_id":               "node-type-id-driver",
 			"task.0.new_cluster.0.node_type_id":               "node-type-id-worker-task",
 			"task.0.new_cluster.0.driver_node_type_id":        "node-type-id-driver-task",
 			"job_cluster.0.new_cluster.0.node_type_id":        "node-type-id-worker-job",
 			"job_cluster.0.new_cluster.0.driver_node_type_id": "node-type-id-driver-job",
 		},
 		HCL: `
-		new_cluster = {
```

Comment:
why is this block removed?

Reply:
Because this config was wrong, it is intending to use the jobs 2.1 API because it has `task` specified, but top level `new_cluster` is never consumed by the 2.1 API, so this `new_cluster` block is never necessary